log.go

```diff
@@ -142,6 +142,12 @@ func setLogLevel(subsystemID string, logLevel string) {
 	logger.SetLevel(level)
 }
 
+// Log writes a message to the log using LevelInfo
 func Log(m string) {
 	log.Info(m)
 }
+
+// LogT writes a tagged message to the log using LevelInfo
```

Comment:
End both comments with period.
```suggestion
// Log writes a message to the log using LevelInfo.
func Log(m string) {
	log.Info(m)
}

// LogT writes a tagged message to the log using LevelInfo.
```